kin-devplatform-sdk/src/main/java/kin/devplatform/ConfigurationLocal.java

```diff
@@ -40,27 +42,31 @@ public static ConfigurationLocal getInstance(@NonNull Context context) {
 
 	@Override
 	public KinEnvironment getEnvironment() {
-		final String blockchainNetworkUrl = configurationSharedPreferences.getString(BLOCKCHAIN_NETWORK_URL_KEY, null);
-		final String blockchainPassphrase = configurationSharedPreferences.getString(BLOCKCHAIN_PASSPHRASE_KEY, null);
+		final String oldBlockchainNetworkUrl = configurationSharedPreferences.getString(OLD_BLOCKCHAIN_NETWORK_URL_KEY, null);
+		final String oldBlockchainPassphrase = configurationSharedPreferences.getString(OLD_BLOCKCHAIN_PASSPHRASE_KEY, null);
+		final String newBlockchainNetworkUrl = configurationSharedPreferences.getString(NEW_BLOCKCHAIN_NETWORK_URL_KEY, null);
```

Comment:
why do we need to keep OLD_XXXX it is not one method that does the migration?

Reply:
the migration is between old a new, you must specify the old network, moreover the SDK will work on kin2 for a while before migrating, so there's a phase where only kin2 is used, and then somewhen migration will happen and only kin3 will be used.